mypyc/ir/rtypes.py

```diff
@@ -234,9 +234,14 @@ def __repr__(self) -> str:
 short_int_rprimitive = RPrimitive('short_int', is_unboxed=True, is_refcounted=False,
                                   ctype='CPyTagged')  # type: Final
 
-# low level integer (corresponds to C's 'int').
-c_int_rprimitive = RPrimitive('c_int', is_unboxed=True, is_refcounted=False,
+# low level integer (corresponds to C's 'int's).
+c_int32_rprimitive = RPrimitive('c_int32', is_unboxed=True, is_refcounted=False,
+                              ctype='int')  # type: Final
+c_int64_rprimitive = RPrimitive('c_int64', is_unboxed=True, is_refcounted=False,
                               ctype='Py_ssize_t')  # type: Final
```

Comment:
See the above comment. (In particular, use `int64_t` as the type.)

Reply:
Actually at commit f3011ad, I've tried to use `int32_t` and `int64_t` just as we discussed. But somehow the above error comes up, I am actually debugging that and once I find out which of this modification leads to that error and resolve it, I'll make sure we have `int32_t` and `int64_t` as the corresponding C types.

Reply:
It looks like you figured this out, good!